src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -875,14 +875,14 @@ final case class EServicesApiServiceImpl(
       JsonObject(
         "name"         -> Json.fromString(eService.name),
         "description"  -> Json.fromString(eService.description),
-        "technology"   -> Json.fromString(eService.technology.toString),
-        "mode"         -> Json.fromString(eService.mode.toString),
+        "technology"   -> Json.fromString(eService.technology.toApi.toString),
+        "mode"         -> Json.fromString(eService.mode.toApi.toString),
         "descriptor"   -> Json.obj(
           "interface"               -> descriptor.interface
             .map(interface =>
               Json.obj("prettyName" -> Json.fromString(interface.prettyName), "path" -> Json.fromString(interface.name))
             )
-            .getOrElse(Json.obj()),
```

Comment:
I would have expected option to be automatically treated as null, therefore without having to use getOrElse, isn't it?

Reply:
> I would have expected option to be automatically treated as null, therefore without having to use getOrElse, isn't it?

Since it's inside a Json.obj it doesn't support to have Option[Json], in fact it gives the following error:

`Required: (String, Json), found: (String, Option[Json])`